client.go

```diff
@@ -48,54 +45,44 @@ var (
 )
 
 const (
-	// DefaultAddress is the default IPV4 address that is used for a connection
-	DefaultAddress = "127.0.0.1"
-	// DefaultPort is the default port used for a connection
-	DefaultPort     = 6640
-	UNIX            = "unix"
-	TCP             = "tcp"
-	SSL             = "ssl"
-	SKIP_TLS_VERIFY = true
+	DefaultTCPAddress  = "127.0.0.1:6640"
+	DefaultUnixAddress = "/var/run/openvswitch/ovnnb_db.sock"
 )
 
-func ConnectDial(protocol string, target string) (net.Conn, error) {
-	if protocol == SSL {
-		cert, err := tls.LoadX509KeyPair(os.Getenv("CLIENT_CERT_CA_CERT"),
-			os.Getenv("CLIENT_PRIVKEY"))
-		if err != nil {
-			log.Fatalf("client: loadkeys: %s", err)
-			return nil, err
-		}
-		if len(cert.Certificate) != 2 {
-			log.Fatal("client.crt should have 2 concatenated certificates: client + CA")
-			return nil, err
+// Connect to ovn, protocol is one of supported by ovn: unix, tcp, ssl
+// If address is empty, use default address for specified protocol
+func Connect(protocol string, address string, tlsConfig *tls.Config) (*OvsdbClient, error) {
```

Comment:
As commented earlier, I think it may be better just:
Connect(connMethod string, tlsConfig *tls.Config)
"connMethod" should be in the format specified by man page of ovsdb(7), section CONNECTION METHODS.